lib/mayaUsd/ufe/private/Utils.cpp

```diff
@@ -128,15 +128,21 @@ void applyCommandRestriction(const UsdPrim& prim, const std::string& commandName
         return;
     }
 
-    auto        primSpec = MayaUsdUtils::getPrimSpecAtEditTarget(prim);
-    auto        primStack = prim.GetPrimStack();
+    SdfPrimSpecHandle       primSpec = MayaUsdUtils::getPrimSpecAtEditTarget(prim);
```

Comment:
Changed to concrete types to make understanding the code easier when reading it.